### PR TITLE
chore: remove personal information

### PR DESCRIPTION
**Pull request by Skarlso**
## Description

Related to https://github.com/open-component-model/ocm-project/issues/43

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

- Related Issue # (issue)
- Closes # (issue)
- Fixes # (issue)
> Remove if not applicable

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help
- [ ] Separate ticket for tests # (issue/pr)

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Added to documentation?

- [ ] 📜 README.md
- [ ] 🙅 no documentation needed

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: pkg/ocm/ocm_test.go [hunk 1]
```diff
@@ -120,9 +120,9 @@ func TestClient_GetComponentVersion(t *testing.T) {
 						".dockerconfigjson": []byte(`{
   "auths": {
     "ghcr.io": {
-      "username": "skarlso",
+      "username": "open-component-model",
       "password": "password",
-      "auth": "c2thcmxzbzpwYXNzd29yZAo="
+      "auth": "b3Blbi1jb21wb25lbnQtbW9kZWw6cGFzc3dvcmQ="
```
Comment (hilmarf) [inline pkg/ocm/ocm_test.go:125]: @Skarlso - is this fake/test data? Or real working credentials? If the second is the case, can we come up with a solution without publishing them?

@morrison-sap f.y.i.

File: pkg/ocm/ocm_test.go [hunk 2]
```diff
@@ -197,9 +197,9 @@ func TestClient_CreateAuthenticatedOCMContext(t *testing.T) {
 			".dockerconfigjson": []byte(`{
   "auths": {
     "ghcr.io": {
-      "username": "skarlso",
+      "username": "open-component-model",
       "password": "password",
-      "auth": "c2thcmxzbzpwYXNzd29yZAo="
+      "auth": "b3Blbi1jb21wb25lbnQtbW9kZWw6cGFzc3dvcmQ="
```
Comment (hilmarf) [inline pkg/ocm/ocm_test.go:202]: credentials!?!

**Review by hilmarf — DISMISSED** [review]

File: pkg/ocm/ocm_test.go (hunk 1 above)
Comment (Skarlso) [reply pkg/ocm/ocm_test.go:125]: It's base64 encoded test data.

File: pkg/ocm/ocm_test.go (hunk 2 above)
Comment (Skarlso) [reply pkg/ocm/ocm_test.go:202]: It's base64 encoded test data.

**Review by hilmarf — APPROVED** [review]

**Pull request merged**